spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="pyload"
+PYTHON="${INSTALL_DIR}/env/bin/python"
+PYLOAD="${INSTALL_DIR}/share/pyload/pyLoadCore.py"
+LOG_FILE="${INSTALL_DIR}/etc/Logs/log.txt"
+PID_FILE="${INSTALL_DIR}/var/pyload.pid"
+
+
+
+start_daemon ()
+{
+    su - ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --daemon"
```

Comment:
Use `su ${USER} -c "PATH=${PATH} ${PYTHON} ${PYLOAD} --pidfile=${PID_FILE} --daemon"` instead (removed the minus sign after 'su')